5001-6000/LIT5696MagicPrayer.xml

```diff
@@ -4,8 +4,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ሐጹረ፡ መስቀል፡</title>
-                <title xml:lang="gez" type="normalized" corresp="#t1">Ḥaṣura Masqal</title>
+                <title xml:lang="en" xml:id="t1">Rampart of the Cross (General record)</title>
+                <title xml:lang="gez" xml:id="t2">ሐጹረ፡ መስቀል፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ḥaṣura Masqal</title>
+                
```

Comment:
I think an abstract would be helpful for clarity and disambiguation, here and elsewhere, as to see the relations (or incipits) one must go to another page, an abstract is what one sees at once

Reply:
I added a short clarification. As I understnd now, there is probably no "standard" main stream tradition of Hasura masqal prayer, or at least it has not been clearly identified yet. Various protective texts referring to Cross and its protection may be called Hasura masqal prayers (also if the words Hasura masqal does not occur there). It cannot be excluded, of course, that a part of those texts build up one tradition (of a specific work called Hasura masqal). So it is a bit vague and we just have to collect further material.